### PR TITLE
Completed the confirmation email functionality

### PR DESCRIPTION
**Pull request by lacf95**
- Email with confirmation email
- Error validation on non exist token

### CODE REVIEW

File: app/controllers/registration_controller.rb [hunk 1]
```diff
@@ -32,7 +32,12 @@ def user_params
...
   end
 
-  def send_email(token)
-    "/#{token}"
+  def send_email
+    url = confirm_url
+    UserNotifierMailer.send_signup_email(@user.id, url).deliver
+  end
+
+  def confirm_url
```
Comment (cesargomez89) [inline app/controllers/registration_controller.rb]: you can use path helper passign the token as parameter

Comment (cesargomez89) [reply app/controllers/registration_controller.rb]: `anything_path(token)`

**Review by cesargomez89 — CHANGES_REQUESTED** [review]
I left 1 comment

**Comment by lacf95** [comment]
Updated now

**Review by cesargomez89 — APPROVED** [review]

**Pull request merged**